doc/sflow/sflow_hld.md

```diff
@@ -887,7 +887,7 @@ On the SONiC VS, SAI calls would map to the tc_sample commands on the switchport
 ## 11 **Restrictions**
 * /etc/hsflowd.conf should not be modified manually. While it should be possible to change /etc/hsflowd.conf manually and restart the sflow container, it is not recommended.
 * configuration updates will necessitate hsflowd service restart
-* hsflowd daemon will initialize only after receiving the SYSTEM_READY|SYSTEM_STATE Status=Up from SONiC. The system ready state however depends on all daemons to be ready and all components like PSU and fan to be functional. Failure on any of these, will result in system state to be down. In such scenarios, sflow will wait until 180 seconds and if system ready is not up will proceed with initialization. For more details please refer to https://github.com/sonic-net/sonic-buildimage/issues/13407#issuecomment-1575257797
+* hsflowd daemon will initialize only after receiving the SYSTEM_READY|SYSTEM_STATE Status=Up from SONiC. The system ready state however depends on all the monitored daemons to be ready. Failure on any of these, will result in system state to be down. In such scenarios, sflow will wait until 180 seconds and if system ready is not up will proceed with initialization. For more details please refer to https://github.com/sonic-net/sonic-buildimage/issues/13407#issuecomment-1575257797 . For system ready feature please visit https://github.com/sonic-net/SONiC/blob/master/doc/system_health_monitoring/system-ready-HLD.md

```

Comment:
IMO, its not recommended to link issues to HLD. These issues can be transferred etc. Since PR already has the description of details, suggest to remove it.

Reply:
Done